time_series_anomaly_detection/univariate_twitter_data_experiments.ipynb

```diff
@@ -0,0 +1,172 @@
+{
```

Comment:
### Important

- I would like to see experiments with real labeled anomalies (for NAB, you get labels from: https://github.com/numenta/NAB/tree/master/labels) (show real anomalies on the plot, compute precision + recall + area under their curve)
- If possible, show experiments on Ozone or Occupancy UCI datasets as in the paper, so that we can compare the results and validate the solution.